README.md

```diff
@@ -404,3 +404,9 @@ Follow our [H2O Stream](https://groups.google.com/forum/#!forum/h2ostream).
     import _root_.hex.tree.gbm.GBM
     ```
   
+* Getting non-deterministic H2O Frames after the Spark Data Frame to H2O Frame conversion.
```

Comment:
Can we explain it little bit based on the comments in https://0xdata.atlassian.net/browse/PUBDEV-3808